src/js/controller/viewMixin/week.js

```diff
@@ -146,17 +147,19 @@ var Week = {
      * @param {Date} start - start date.
      * @param {Date} end - end date.
      * @param {Collection} time - view model collection.
+     * @param {Object} options - week view options
      * @returns {object} view model for time part.
      */
-    getViewModelForTimeView: function(start, end, time) {
+    getViewModelForTimeView: function(start, end, time, options) {
```

Comment:
Make weekly options object separate with this module. This function doesn't need to know all options of weekly, just only to know hourStart, hourEnd.